src/Bot.ts

```diff
@@ -91,11 +93,24 @@ export class Bot extends builder.UniversalBot {
         };
     }
 
+    private async verifySigninState(
+        event: builder.IEvent,
+        query: teams.ISigninStateVerificationQuery,
+        callback: (err: Error, body: any, status?: number) => void): Promise<void>
+    {
+        let session = await loadSessionAsync(this, event);
+        if (session)
+        {
+            session.clearDialogStack();
+            session.send(Strings.popupsignin_successful);
```

Comment:
For this example, let's pass the user's input username and password through the notifySuccess call so that the verifyState handler in the bot can present those fields in the outgoing message - so here, the handler should grab the passed in values and send those out in the message.